### PR TITLE
Changelog 2.7.0

### PR DESCRIPTION
**Pull request by rami-elementor**

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -90,6 +90,35 @@ Source: https://stocksnap.io/photo/4B83RD7BV9
...
+* Tweak: Escape translation strings for secure HTML output
+* Tweak: Use i18n function to make the "Menu" string translatable
+* Tweak: Minify SVG assets
+* Tweak: Make header nav-menu keyboard accessible
+* Tweak: Add `role="button"` to the nav-menu toggle for better accessibility
+* Tweak: Toggle mobile nav-menu with `Enter` & `Space` keyboard keys
+* Tweak: Add `hello_elementor_enable_skip_link` filter to enable/disable the skip link
+* Tweak: Add `hello_elementor_skip_link_url` filter to change skip link URL
+* Tweak: Use theme CSS not Elementor plugins CSS
+* Tweak: Match CSS to the new Elementor Editor Facelift
```
Comment (arielk) [inline readme.txt:109]: Remove this line (duplicate)

File: readme.txt [hunk 2]
```diff
@@ -90,6 +90,35 @@ Source: https://stocksnap.io/photo/4B83RD7BV9
...
+* Tweak: Added support for the new Elementor version
+* Tweak: Match autoprefixer browsers list settings to Elementor plugin
```
Comment (arielk) [inline readme.txt:111]: Remove this line (not relevant)

File: readme.txt [hunk 3]
```diff
@@ -90,6 +90,35 @@ Source: https://stocksnap.io/photo/4B83RD7BV9
...
+* Tweak: Move autoprefixer settings to `.browserslistrc` file
```
Comment (arielk) [inline readme.txt:112]: Remove this line (not relevant)

File: readme.txt [hunk 4]
```diff
@@ -90,6 +90,35 @@ Source: https://stocksnap.io/photo/4B83RD7BV9
...
+* Tweak: Upgrade the autoprefixer package to minify CSS files
```
Comment (arielk) [inline readme.txt:113]: Remove this line (not relevant)

**Review by arielk — CHANGES_REQUESTED** [review]

File: readme.txt (hunk 2 above)
Comment (rami-elementor) [reply readme.txt:111]: Replaced with:
* Tweak: Update autoprefixer to exclude dead browsers

File: README.md [hunk 5]
```diff
@@ -92,6 +92,26 @@ Source: https://stocksnap.io/photo/4B83RD7BV9
 
 ## Changelog ##
 
+### 2.7.0 - 2023-03-21 ###
+* Tweak: Update `Requires at least: 5.9`
+* Tweak: Update `Tested Up to 6.1`
```
[the remark this answers is not in the document]
Comment (rami-elementor) [reply README.md]: Done

File: readme.txt (hunk 3 above)
Comment (rami-elementor) [reply readme.txt]: Done

File: readme.txt (hunk 4 above)
Comment (rami-elementor) [reply readme.txt]: Done

File: readme.txt (hunk 1 above)
Comment (rami-elementor) [reply readme.txt]: Done

**Review by arielk — APPROVED** [review]

**Pull request merged**